packages/features/eventtypes/components/WeightDescription.tsx

```diff
@@ -0,0 +1,18 @@
+import type { TFunction } from "i18next";
+import Link from "next/link";
+
+import CustomTrans from "@calcom/web/components/CustomTrans";
+
+export default function WeightDescription({ t }: { t: TFunction }) {
+  return (
```

Comment:
just for curiosity, any particular reason why you're passing `t` here instead of getting it from useLocale()?